inc/class-scripts.php

```diff
@@ -335,7 +335,27 @@ public function register_default_styles(): void {
 	 */
 	public function enqueue_default_admin_styles(): void {
 
-		wp_enqueue_style('wu-admin');
+		// Get current screen to conditionally load styles
+		$screen = get_current_screen();
+
+		if (!$screen) {
+			return;
+		}
+
+		// Only load styles on WP Ultimo admin pages
+		if ($this->is_wp_ultimo_admin_page($screen)) {
+			wp_enqueue_style('wu-admin');
+
+			// Load flag styles only when needed (e.g., on pages with country selection)
+			if ($this->is_wp_ultimo_flags_page($screen)) {
+				wp_enqueue_style('wu-flags');
+			}
+		}
+
+		// Load checkout styles only on checkout pages
+		if (isset($_GET['page']) && strpos($_GET['page'], 'wu-checkout') === 0) {
```

Comment:
please use str_starts_with instead. the polyfill is included in vendor so it will always work even on PHP 7.4